front/views/AuthenticationView.tsx

```diff
@@ -3,18 +3,54 @@ import { useDispatch } from '../state/Store';
 import { translate } from "../i18n/i18n";
 import { setUserToken } from "../state/UserSlice";
 import { Center, Button, Text } from 'native-base';
+import SigninForm from "../components/forms/signinform";
+import SignupForm from "../components/forms/signupform";
+
+const checkLoginCredentials = async (username: string, password: string, tokenSetter: (token: string) => void) => {
+	return new Promise<string>((resolve, reject) => {
+		setTimeout(() => {
+			if (username === "katerina" && password === "1234") {
+				// got a red warning if i don't wait a little bit before setting the token 
+				setTimeout(() => tokenSetter("1234"), 500);
+				return resolve("st germain c'est stylé, big up à Arthur");
+			}
+			return reject(translate("invalidCredentials"));
+		}, 1000);
+	});
+};
+
+const checkSignupCredentials = async (username: string, password: string, email: string, tokenSetter: (t: string) => void) => {
+	return new Promise<string>((resolve, reject) => {
+		setTimeout(() => {
```

Comment:
Instead of authenticating that way, for an easier integration, there should be a method in the api wrapper that manages this. Of course, for now it should do just the same thing